frontend/src/components/MapView/MapTooltip/PopupCharts.tsx

```diff
@@ -0,0 +1,246 @@
+import { appConfig } from 'config';
+import { BoundaryLayerProps } from 'config/types';
+import {
+  getBoundaryLayersByAdminLevel,
+  getWMSLayersWithChart,
+} from 'config/utils';
+import { BoundaryLayerData } from 'context/layers/boundary';
+import { LayerData } from 'context/layers/layer-data';
+import {
+  layerDataSelector,
+  layersSelector,
+} from 'context/mapStateSlice/selectors';
+import React, { memo, useCallback, useEffect, useRef, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faExternalLinkAlt, faTimes } from '@fortawesome/free-solid-svg-icons';
+import {
+  Button,
+  IconButton,
+  WithStyles,
+  createStyles,
+  withStyles,
+} from '@material-ui/core';
+import { MapTooltipState } from 'context/tooltipStateSlice';
+import i18n, { isEnglishLanguageSelected } from 'i18n';
+
+import ChartSection from '../LeftPanel/ChartsPanel/ChartSection';
+import { oneYearInMs } from '../LeftPanel/utils';
+import DownloadCsvButton from '../DownloadCsvButton';
+import { buildCsvFileName } from '../utils';
+
+const chartLayers = getWMSLayersWithChart();
+const { countryAdmin0Id, country, multiCountry } = appConfig;
+const MAX_ADMIN_LEVEL = appConfig.multiCountry ? 3 : 2;
+const admin0Offset = multiCountry ? 0 : 1;
```

Comment:
It might make more sense to use something like this instead of these two variables? wdyt @jbperidypathtech ?
`const availableAdminLevels = multiCountry ? [0, 1, 2] : [1, 2];`